django/db/backends/sqlite3/creation.py

```diff
@@ -142,9 +142,12 @@ def setup_worker_connection(self, _worker_id):
             connection_str = (
                 f"file:memorydb_{alias}_{_worker_id}?mode=memory&cache=shared"
             )
-            source_db = self.connection.Database.connect(
-                f"file:{alias}_{_worker_id}.sqlite3?mode=ro", uri=True
-            )
+            try:
+                source_db = self.connection.Database.connect(
+                    f"file:{alias}_{_worker_id}.sqlite3?mode=ro", uri=True
+                )
+            except sqlite3.OperationalError:
+                return
```

Comment:
This change should no longer be necessary. Can you revert?

Reply:
Sure 👍  Would be more than happy to since it makes the code harder to read 😛  (I left it in, just in case you wanted to keep some defensive measures in there)